coderedcms/blocks/content_blocks.py

```diff
@@ -12,6 +12,19 @@
 from .html_blocks import ButtonBlock
 
 
+class AccordionBlock(ButtonMixin, BaseBlock):
```

Comment:
In this case, I would prefer to not use ButtonMixin, only because things like setting page, url, doc links are not relevant. Might be better off to inline a `title` field on this model and keep it simple.

Reply:
In this case, all the button mixin does is add the three fields for the title, style, and size. Button Link is a different thing: 
![image](https://user-images.githubusercontent.com/30885417/180839190-6995f653-aa31-4e83-9cc2-523e4c94167e.png)

Reply:
Please take away the button mixin. This should function exactly like the carousel where they choose one Accordion snippet, and it renders the whole thing.